# HG changeset patch
# User Fabian Groffen <grobian@gentoo.org>
# Date 1522498552 -7200
#      Sat Mar 31 14:15:52 2018 +0200
# Node ID 2b03f35544c42abf3ff3c0931ec355ac9fb2ca57
# Parent  891c47134b3d8c89089a21bbf703d6e7562a7538
cmd_handle_fatal: make error message a bit more descriptive

When there are multiple IMAP connections available, "Mailbox closed"
doesn't give a hint as to which one. Use account info to identify which
mailbox was closed.

https://gitlab.com/muttmua/mutt/merge_requests/3

diff -r 891c47134b3d -r 2b03f35544c4 imap/command.c
--- a/imap/command.c	Sat Mar 31 14:15:52 2018 +0200
+++ b/imap/command.c	Sat Mar 31 14:15:52 2018 +0200
@@ -486,7 +486,8 @@ static void cmd_handle_fatal (IMAP_DATA*
   {
     mx_fastclose_mailbox (idata->ctx);
     mutt_socket_close (idata->conn);
-    mutt_error (_("Mailbox closed"));
+    mutt_error (_("Mailbox %s@%s closed"),
+	idata->conn->account.login, idata->conn->account.host);
     mutt_sleep (1);
     idata->state = IMAP_DISCONNECTED;
   }
